image/src/integrations/dbt/__init__.py

```diff
@@ -0,0 +1,22 @@
+import os
+from dataclasses import dataclass
+
+from grai_source_dbt import base
+
+
+@dataclass
+class Args:
+    manifest_file: str = os.environ["GRAI_DBT_MANIFEST_FILE"]
+    namespace: str = os.environ.get("GRAI_NAMESPACE", "default")
```

Comment:
Do we need to default here if the action has a default?  I note we haven't got a default in image/src/main.py

Reply:
Need is a strong word - I don't think it's needed for the code to function (though it doesn't introduce any bugs either). The reason I included a default here was a signal to future viewers that the value was optional and to treat it as such. I don't